docs/indexing-guide/multidimensional-indices/boolean-arrays.md

```diff
@@ -429,7 +429,7 @@ saturation of only those pixels:
 .. plot::
    :context: close-figs
    :include-source: True
-   :output-base-name: astronaut-{counter}
+   :image-basename: astronaut-saturated-better
```

Comment:
Can you move the changes to this file to its own PR

Reply:
This must be coming from a bad rebase or something. This change is from another PR and is already present in main.

Reply:
Yup, my bad, fixed.